lib/dotcom/schedule_finder.ex

```diff
@@ -46,6 +51,51 @@ defmodule Dotcom.ScheduleFinder do
           }
   end
 
+  @doc """
+  Service-impacting currently active alerts at a stop/route/direction. This also
+  includes track changes, and excludes commuter rail trip cancellations and
+  delays.
+  """
+  @spec current_alerts(Route.t(), 0 | 1, Stop.id_t()) :: [Alert.t()]
+  def current_alerts(route, direction_id, stop_id) do
+    stop_alerts = @alerts_repo.by_stop_id(stop_id)
+
+    route_alerts =
+      @alerts_repo.by_route_id_and_type(route.id, route.type, @date_time_module.now())
+
+    relevant_entity =
+      InformedEntity.from_keywords(
+        route_type: route.type,
+        route: route.id,
+        stop: stop_id
+      )
```

Comment:
<img width="1824" height="718" alt="Image" src="https://github.com/user-attachments/assets/e7be07a2-f10b-4409-b5a9-4d38275d7777" />

I think this is too strict, which is causing relevant route-wide alerts not to show up, except on the stops that they directly impact.


```suggestion
    relevant_entities =
      [
        InformedEntity.from_keywords(route: route.id),
        InformedEntity.from_keywords(route_type: route.type),
        InformedEntity.from_keywords(stop: stop_id),
      ]
```

Changing it like so ☝️ looks like it effectively changes the route/stop/mode criteria from an AND to an OR, which I think more closely maps to the behavior we want.